Enterprise-CMCS/macpro-mdct-mfp
# MDCT-2956: Static Target Populations w/ Dynamic "Other" Entities

### Description

Switching over the hard-coded target populations to the static initial 4 entities prior to adding "Other" target populations.

### Related ticket(s)


MDCT-2956

---
### How to test

- Go to the Transition Benchmarks page under the Work Plan
- You should see four (4) initial entities and be able to add "Other" entities under them

### Important updates



---
### Author checklist


---


_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_
